requirements.txt

```diff
@@ -1,2 +1,8 @@
 llama-index==0.9.29
```

Comment:
If we go with langchain, we don't need llama-index for now. Think we can get rid of the dependency.